packages/vite/src/node/importGlob.ts

```diff
@@ -1,3 +1,4 @@
+import { isCSSRequest } from './plugins/css'
```

Comment:
I think we should move this import down a bit, beside the `./plugins/importAnalysisBuild` import perhaps